setup.cfg

```diff
@@ -35,12 +35,14 @@ setup_requires =
     pytest-runner
 install_requires =
     makefun>=1.5.0
+    typing_extensions;python_version>='3.6'
     # note: do not use double quotes in these, this triggers a weird bug in PyCharm in debug mode only
     funcsigs;python_version<'3.3'
     enum34;python_version<'3.4'
 tests_require =
     pytest
     pytest_cases
+    syrupy;python_version>'3.6'
```

Comment:
Could you please add a comment above this line explaining why you declare syrupy and not pyright ?